web-client/src/presenter/actions/getCaseTypesAction.js

```diff
@@ -1,5 +1,13 @@
 import { state } from 'cerebral';
 
+/**
+ *  Gets a list of the types of cases, such as Deficiency, Innocent Spouse, etc
+ *
+ * @param {Object} providers the providers object
+ * @param {Object} providers.applicationContext needed for getting the getCaseTypes use case
+ * @param {Functionn} providers.get the cerebral get function used for getting state.user.userId
```

Comment:
I'm seeing a few of these `Functionn`, so you might want to just do a quick find-and-replace.